c2corg_ui/static/js/documentediting.js

```diff
@@ -249,6 +249,10 @@ app.DocumentEditingController.prototype.successRead = function(response) {
   if (this.modelName === 'image') {
     data['date_time'] = new Date(data['date_time']);
   }
+  // xreports date in editing form
```

Comment:
Is it not possible to add this to the ``XreportEditingController``?
If yes, since we are a bit in a hurry, that could be done later (to add to the todo list?).